shell/install-dart-sdk.sh

```diff
@@ -17,9 +14,15 @@ wget http://storage.googleapis.com/dart-archive/channels/stable/release/latest/s
 
 echo "Downloading done."
 
+# remove the currently install version of the dart sdk
+rm -r /usr/local/bin/dart-sdk
+
 echo "Unzipping to /usr/local/bin"
 unzip dartsdk-linux-x64-release.zip -d /usr/local/bin
 
+# for some reason permissions default to 700
+chmod -R 755 /usr/local/bin/dart-sdk/
```

Comment:
I had to add this because the permissions for the `/usr/local/bin/dart-sdk/` directory was being set to 700. dunno what's happening there.